### PR TITLE
Fix for #456

### LINKED ISSUE #456: Decode<JwtHeader> not working
### Regression issue in Decode
When upgrading from 9.0.3 to 10.0.0 decode header will cease to function correctly, all fields will be `null`:

```csharp
var tokenString = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJzdWIiOiIxMjM0NTY3ODkwIiwibmFtZSI6IkpvaG4gRG9lIiwiaWF0IjoxNTE2MjM5MDIyfQ.SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c";
var serializer = new JsonNetSerializer();
var urlEncoder = new JwtBase64UrlEncoder();
var decoder = new JwtDecoder(serializer, urlEncoder);

var header = decoder.DecodeHeader<JwtHeader>(tokenString);
Console.WriteLine(header.Algorithm);
```
For 9.0.3
```
HS256
```
For 10.0.0
```
 // Empty string or null
```

### PR DESCRIPTION
**Pull request by hartmark**
Fixes #456.

### CODE REVIEW

**Comment by hartmark** [comment]
This will fix #456

File: src/JWT/JWT.csproj [hunk 1]
```diff
@@ -13,6 +13,9 @@
   <PropertyGroup Condition="'$(TargetFramework)' == 'net6.0'">
     <TargetFrameworkIdentifier>.NETFramework,Version=6.0</TargetFrameworkIdentifier>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(TargetFramework)' == 'net7.0'">
```
Comment (abatishchev) [inline src/JWT/JWT.csproj:16]: Let's add 7.0 (if we decide so) in a separate PR, see also #451.

**Review by abatishchev — CHANGES_REQUESTED** [review]

**Comment by abatishchev** [comment]
Thanks a lot, @hartmark! Can you please rename the PR to give a bit more readable title?

File: src/JWT/JWT.csproj [hunk 2]
```diff
@@ -13,6 +13,9 @@
   <PropertyGroup Condition="'$(TargetFramework)' == 'net6.0'">
```
Comment (abatishchev) [inline src/JWT/JWT.csproj:13]: And please bump the version to 10.0.2 so I'll push a fix asap.

**Review by abatishchev — CHANGES_REQUESTED** [review]

**Comment by hartmark** [comment]
Replaced by  #463  and  #464

**Pull request closed without merging**